dataloader/utils.py

```diff
@@ -55,11 +55,11 @@ def should_download_file(file_lang: str, file_name: str) -> bool:
     Limit source file set size to speed up loading process
     Can be controlled with the `LIMIT` environment variable.
     """
-    if file_lang == LANG_PALI:# and file_name.startswith("dn10"):
+    if file_lang == LANG_PALI:
         return True
-    elif file_lang == LANG_CHINESE:# and file_name.startswith("T01_T0030"):
+    elif file_lang == LANG_CHINESE:
         return True
-    elif file_lang == LANG_TIBETAN:# and file_name.startswith("T06TD4021"):
+    elif file_lang == LANG_TIBETAN:
```

Comment:
hmm, so we're disabling the testing limit?

Reply:
I agree, better to stick to the TESTING_LIMIT environment variable, please disregard this.

Reply:
In that case, can we re-add the `file_name` checks here?